packages/devtools-reps/src/object-inspector/utils/load-properties.js

```diff
@@ -76,6 +79,10 @@ function loadItemProperties(
     loadingPromises.push(enumSymbols(getObjectClient(), start, end));
   }
 
+  if (shouldLoadFullTextProperty(item, loadedProperties)) {
+    loadingPromises.push(getLongStringFullText(getValue(item)));
```

Comment:
This is def not ready yet, I need to pass the `getlongStringFullText(stringGrip)` redux bound action to `loadItemProperties()`.

Reply:
I did not realized you were looking at expanding longStrings as well, this is great !
Maybe we should this PR focused on merging longString and string rep, and you can move the work on the object inspector on a dedicated PR ?

Reply:
Is there a way to expose `WebConsoleClient.getString` by passing in the longString's object client, as we're doing for other properties, and then calling `objectClient.getString` from there? This way I'll be able to add a `getString` util fn in `object-inspector/utils/client.js`, and structure it more consistently. 💪

Reply:
@nchevobbe  Whoops just noticed ur comment, uum sure thing!